### PR TITLE
chords: Suggest some tweaks

### PR DESCRIPTION
**Pull request by tjvr**
I went through the chords and noted where the chords didn't seem to match what I expected. I particularly looked at some places where the timing didn't seem to match (i.e. the chord seemed to be next to the wrong word). It would be good to make sure we agree on this

Please treat these as suggestions; I may well be wrong! Feel free to either a) pick selected changes from this as you desire or b) note anything you disagree with, and I'll change it back. :blush:

### CODE REVIEW

File: chords/in_christ_alone.onsong [hunk 1]
```diff
@@ -2,6 +2,10 @@
 {artist:  Keith Getty / Stuart Townsend}
 {key: D}
 
+intro:
+|[A].   .   .   |[G].   .   .   |[D].   .   [G].   |[D].   .   .   |
```
Comment (tjvr) [inline chords/in_christ_alone.onsong:6]: I don't know if we want to use this instrumental bit between verses or not, but if we do, then we might as well include it!

File: chords/once_again.onsong [hunk 2]
```diff
@@ -27,9 +27,5 @@ And [G]I'm full of [A]praise once a[D]gain
 
 Bridge
 [G]Thank you for the [A]cross
-[G]Thank you for the [A]cross
-[G]Thank you for the [A]cross, my [D]friend
-
-[Bm]Thank you for the [C#m]cross
-[A]Thank you for the [D]cross
+[D/F#]Thank you for the [Bm]cross
```
Comment (tjvr) [inline chords/once_again.onsong:30]: `[Bm]Thank` sounded really odd to me here. My suggestion here is from [Worship Together](https://www.google.com/search?client=ubuntu&channel=fs&q=worship+together+once+again&ie=utf-8&oe=utf-8)

File: chords/when_the_music_fades.onsong [hunk 3]
```diff
@@ -5,10 +5,10 @@
 
 Verse 1:
 [D]When the music [A]fades
-All is stripped [Em/G]away
+All is stripped [Em]away
```
Comment (tjvr) [inline chords/when_the_music_fades.onsong]: Worship Together just uses `Em` here; `Em/G` didn't sound right to me (but I could be wrong!)

Comment (jdfergusson) [reply chords/when_the_music_fades.onsong]: I think it should work with the bass guitar. They can be ignored if needs be.

Comment (tjvr) [reply chords/when_the_music_fades.onsong]: :heavy_check_mark: Reverted!

**Review by jdfergusson — APPROVED** [review]
LGTM

**Pull request merged**